Add python section to sign-into-web-app guide. OKTA-354778 (#1897)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/require-authentication/python/reqautheverything.md

> In order to handle auth for all routes within a group, you need to use some middleware or use `@before_request` decorator:

Reviewer: ```suggestion
To handle auth for all routes within a group, you need to use some middleware or use the `@before_request` decorator:
```
